src/utils/preprocessing.py

```diff
@@ -394,3 +395,15 @@ def pcd_processing_gapnet(pcd_paths):
     pointclouds = pointclouds.reshape((-1, 1024, 3))
 
     return pointclouds
+
+
+def posenet_processing(filename):
+    image = cv2.imread(filename)
+    if '_100_' in filename or '_101_' in filename or '_102_' in filename:
```

Comment:
Put this in dict. For your ref https://github.com/Welthungerhilfe/cgm-ml/blob/4c802ea9ca1fb666f0d51b090f1524d3d4d39c1d/src/analyses/reliability/Standing-laying/utils.py#L64

Reply:
updated the code